app/src/main/java/org/jellyfin/androidtv/TvApp.java

```diff
@@ -197,11 +199,7 @@ public boolean canManageRecordings() {
     }
 
     public Drawable getDrawableCompat(int id) {
```

Comment:
Better option is to just remove this function from the TvApp class

Reply:
Agreed. Wasn't sure if it belonged in this PR though since I started out just removing stuff and this is used _everywhere_. :rofl:

Reply:
Right, I'll merge this one then